docs/README.md

```diff
@@ -44,6 +44,7 @@
     * [Take Screenshot](articles/documentation/test-api/actions/take-screenshot.md)
     * [Upload](articles/documentation/test-api/actions/upload.md)
     * [Resize Window](articles/documentation/test-api/actions/resize-window.md)
+    * [Debug](articles/documentation/test-api/actions/debug.md)
```

Comment:
I'd rather move it out of actions sections, since it's not action like e.g. navigation